components/lib/crash/src/main/AndroidManifest.xml

```diff
@@ -17,6 +18,10 @@
         <service android:name=".handler.CrashHandlerService"
             android:process=":mozilla.components.lib.crash.CrashHandler"
             android:exported="false" />
+
+        <service android:name=".service.SendCrashReportService"
+            android:process=":mozilla.components.lib.crash.CrashHandler"
```

Comment:
nit: Let's not define a process here. The `CrashHandlerService´ needs that to escape from the crashing process. The service for sending does not have this constraint.

Reply:
agreed.  Thanks,